server/modules/survey/service/publish/surveyPublishJob.js

```diff
@@ -24,6 +25,7 @@ export default class SurveyPublishJob extends Job {
       new SurveyInfoValidationJob(),
       new CyclesDeletedCheckJob(),
       new ProcessingChainsCyclesCheckJob(),
+      new ChainsValidationJob(),
```

Comment:
should this go before ProcessingChainsCyclesCheckJob?

Reply:
uhm, that job removes chains that won't belong to any cycles, it cleans up the chains.

Reply:
right!